crates/building_blocks_storage/src/vox_format_conversions.rs

```diff
@@ -0,0 +1,148 @@
+use std::{fs::File, io::{Cursor, Read, Seek}, path::Path};
+
+use crate::{
+    Array3x1,
+    GetMut,
+};
+use vox_format::{
+    data::{VoxBuffer, VoxModelBuffer}, 
+    reader::{Error, read_vox_into}, 
+    types::{Color, ColorIndex, Palette, Point, Size, Voxel},
+};
+
+use building_blocks_core::{Extent3i, Point3i, PointN};
+    
+/// Re-export `vox_format` crate.
+pub use vox_format;
+
+
+/*
+
+These must live in building_blocks_core`
```

Comment:
Yea you ought to be able to have these in `building_blocks_core`. Is there a problem with that?

Reply:
No, if that's okay, I will do that. I just wanted to minimize the number of places this touches.

Reply:
Works for me. There are already features for nalgebra and glam, so you can follow the pattern there.

Reply:
done